contracts/extensions/CoinMachine.sol

```diff
@@ -30,7 +30,7 @@ contract CoinMachine is ColonyExtension, BasicMetaTransaction {
 
   // Events
 
-  event TokensBought(address buyer, uint256 numTokens, uint256 totalCost);
+  event TokensBought(address indexed buyer, address indexed token, uint256 numTokens, uint256 totalCost);
```

Comment:
I can understand indexing `buyer`, but I'm struggling to come up with a justification for indexing `token` here?

Reply:
Fair point, given that there is only one value this argument can take. I can remove.